File: Python Code/Daily_Flash_Code_0/Week 3/31Jan/16-DailyFlash_Solutions/16-DailyFlash_Solutions/31_Jan_Solutions_Two/CPP/progam5.cpp

#include <iostream>

int main () {

	int n, j, sum = 0;
	std::cout << ("Enter Limit : ");
	std::cin >> n;

	if (n < 0)
		exit(0);

	for(int i = 1; i <= n; i++){
		sum = 0;
		for(j = 1; j <= i/2; j++){
			
			if(i % j == 0){
				sum = sum + j;
			}
		}
		if(sum == i)
			std::cout << i << " ";
	}
	std::cout << ("\n");
}

File: Python Code/Daily_Flash_Code_0/Week 3/31Jan/16-DailyFlash_Solutions/16-DailyFlash_Solutions/31_Jan_Solutions_Two/Python/progam5.py
try:
	n = int(input("Enter Limit "))
except ValueError as e:
	exit(0)	
if(n <= 0):
	exit(0)

for i in range(1, n + 1):
	sum = 0
	for j in range(1, i//2 + 1):
			
		if(i % j == 0):
			sum = sum + j
	if(sum == i):
		print(i, end =  ' ')

print()
